design/src/production-chains/index.md

```diff
@@ -0,0 +1,137 @@
+# Production chains
+
+In a [factory builder](../high-level/genre-mechanics.md), resources are extracted, transformed and ultimately used.
+They are extracted as [raw resources](raw-resources.md), are transformed into [intermediates](intermediates.md) are ultimately turned into useful [end products](end-products.md).
+
+The paths that these **items** can travel on is dictated by the [recipes](recipes.md) available to the player.
+
+## Analyzing production chains
+
+When designing production chains for a factory builder (or frankly, any game with crafting),
+the most essential thing to consider is the **recipe graph.**
+
+You can construct the recipe graph by placing the items in the game as nodes of the graph, and then drawing an edge from each of the inputs in a recipe to its outputs.
+We can also **color** this recipe graph into three parts that roughly correspond to raw resources, intermediates and end products.
+Raw resources have no parents, end products have no children*, and intermediates have both.
+
+As an aside, these are actually directed hypergraphs.
+Hypergraphs may sound scary, but they're pretty simple: they're simply graphs that connect **nodes** to each other via **edges**, but the edges map betweeen groups of nodes rather than individual nodes.
+"Directed" simply means that the edges have a direction to them: you can't turn electricity back into coal!
+
+Unsurprisingly, this gets quite intimidating for most games!
+Instead, it's more useful to:
+
+1. Consider smaller parts of the graph at a time.
+2. Reason abstractly about the properties of these graphs, and their effect on gameplay.
+
+Let's go through some of the relevant properties of production graphs!
+
+### Number of nodes (vertex count)
+
+This is a simple measure of the complexity of the game's crafting system.
+If there are more items to find and make, there's more game to both learn and play.
+
+Increasing this value increases playtime and diversity at the cost of complexity and development costs.
+
+### Number of [connected components](https://en.wikipedia.org/wiki/Component_(graph_theory))
+
+Imagine the recipes as physically connecting various parts of your graph, and then try to pull it apart.
+How many parts can you split it into without breaking any of the edges?
+
+The number of connected components is *very* high relative to the number of nodes in most farming games, survival crafters and other simple crafting systems.
```

Comment:
This kind of makes it sounds like the number of connected components is higher than the number of nodes, which is not possible.

Also, I think we should add how this relation is in most Factory builders. I assume you usually have a very small number of connected components?